app/helpers/user_membership_helper.rb

```diff
@@ -0,0 +1,24 @@
+module UserMembershipHelper
+  def membership_icons(user)
+    UserMembership::ALL_MEMBERSHIPS.reduce("".html_safe) do |acc, elem|
+      acc + " " + membership_icon(user, elem)
+    end
+  end
+
+  def membership_icon(user, membership)
+    if user.show_membership?(membership)
+      image = "membership/#{membership}"
+      alt = t("user.view.membership.#{membership}")
+      title = t("user.view.membership.#{membership}")
+      id = "badge-membership-#{membership}"
+    end
+
+    if image
```

Comment:
This will always evaluate to true, if the `user.show_membership?` evaluated to true. So I think it would be better to combine the two if statements

Reply:
Oh, I see now this is done this way to match the user_roles_helper logic, which is quite similar.